src/TestFixture.cc

```diff
@@ -210,6 +210,13 @@ TestFixture &TestFixture::OnPostUpdate(std::function<void(
 //////////////////////////////////////////////////
 std::shared_ptr<Server> TestFixture::Server() const
 {
+  if (!this->dataPtr->finalized)
+  {
+    ignwarn << "Fixture has not been finalized, any functions you attempted"
```

Comment:
Can we use `gzwarn` instead to make forward porting easier?

Reply:
I don't think `gzwarn` is available in citadel. It might be worth considering back porting the functionality to reduce the maintenance overhead.

Reply:
It's not.  If we go ahead and do the forward port while it's "fresh" we can minimize some pain

Reply:
Ah, sorry. For some reason I thought they were. @mjcarroll would there be an issue if we added those to gz-common3 at some point?